.env.example

```diff
@@ -1,16 +0,0 @@
-# For Sanity Studio Locally
```

Comment:
This file should remain as an example, but it needs updating with the Dato-specific token